packages/ymax-planner/src/engine.ts

```diff
@@ -395,15 +448,32 @@ export const startEngine = async ({
           return;
         }
 
-        // Filter for ymax portfolio paths.
+        // Filter for paths we care about (portfolios or subscriptions).
         const path = encodedKeyToPath(attributes.key);
-        if (!vstoragePathStartsWith(path, VSTORAGE_PATH_PREFIX)) return;
+        if (
+          !vstoragePathStartsWith(path, VSTORAGE_PATH_PREFIX) &&
+          !vstoragePathStartsWith(path, ORCHESTRATION_SUBSCRIPTIONS_PREFIX)
+        )
+          return;
```

Comment:
Suggestion: Use fewer prefix checks.
```suggestion
        if (vstoragePathStartsWith(path, VSTORAGE_PATH_PREFIX)) {
          return { type: 'portfolio', path, value: attributes.value };
        }
        if (vstoragePathStartsWith(path, ORCHESTRATION_SUBSCRIPTIONS_PREFIX)) {
          return { type: 'subscription', path, value: attributes.value };
        }
```